frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -270,57 +271,209 @@
 
 ;; ---- Components box ----
 
+(def components-to-group (atom []))
```

Comment:
Those new functions should be in the "Groups assets management" section, at the top of the file

Reply:
Fixed